netket/logging/hdf5_log.py

```diff
@@ -0,0 +1,179 @@
+import os
+import numpy as np
+from ..logging import RuntimeLog
+from flax.serialization import to_bytes
+
+try:
+    import h5py
+except ImportError:
+    pass
+
+
+def _exists_hdf5(prefix):
+    return os.path.exists(prefix + ".hdf5")
```

Comment:
Nitpick: Since this function is only used once, can you actually not define it and just paste it's content where you use it?